Content.Server/StationEvents/Events/SolarFlare.cs

```diff
@@ -0,0 +1,60 @@
+using Content.Server.Radio.Components;
+using Content.Server.Radio;
+
+namespace Content.Server.StationEvents.Events
+{
+    public sealed class SolarFlare : StationEventSystem
+    {
+        public override string Prototype => "SolarFlare";
+
+        private const string affectedChannel = "Common";
+
+        private float _endAfter = 0.0f;
+        private bool _running = false;
```

Comment:
can you get rid of this and check if `Elapsed > 0`  when needed instead?

Reply:
Elapsed is only reset when event is Added, so this won't work unless I zero Elapsed when event ends